Tests/API/BasicAPITest.php

```diff
@@ -42,10 +50,10 @@ public function testAuthenticated()
             $content = json_decode($result['content']);
             $response = $result['response'];
 
-            $this->assertTrue(empty($result['error']));
-            $this->assertTrue(!empty($content));
-            $this->assertTrue(!empty($content->location));
-            $this->assertTrue($response == 200);
+            $this->assertTrue(empty($result['error']), 'The result should not contain an error property.');
```

Comment:
I think there's an assertEmpty() and an assertArrayHasKey which might help with the error messages.